week1/project/EcommerceVika/ecommerce-shop-project/src/components/Products.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./products.css";
+
+
+function Products({filteredProducts}) {
+  return (
+    <div className="products">
+        {filteredProducts.map((product) => (
+          <div key={product.id} className="product">
+            <img src={product.image} alt={product.title} />
+            <div className="product-details">
+              <div className="product-title">{product.title}</div>
+              <div className="product-price">{product.price}</div>
+              <div className="product-description">{product.description}</div>
+              <div className="product-category">{product.category}</div>
+              <div className="product-rating">
+                Rating: {product.rating.rate} ({product.rating.count} reviews)
+              </div>
+            </div>
+          </div>
```

Comment:
this would be great if separated in a new component

Reply:
You mean to make product.js with own css file? Right?